src/controllers/baseHandler.ts

```diff
@@ -13,7 +13,8 @@ import {
   messageRequestDataOptions,
 } from "../typeDefinitions/discordMessage.types";
 
-import { HELLO, MENTION_EACH, VERIFY } from "../constants/commands";
+import { HELLO, MENTION_EACH, OOO, VERIFY } from "../constants/commands";
+import { oooCommand } from "./oooCommand";
```

Comment:
This import should be before constants maintaining the import structure of the project. (personal opinion)

Reply:
To rectify the import order, we have the option of utilizing **eslint** to establish rules.

Reply:
sure. will make the change.